pom.xml

```diff
@@ -89,10 +89,10 @@
             <artifactId>xembly</artifactId>
             <version>0.21</version>
         </dependency>
-        <dependency>
-            <groupId>com.jcabi</groupId>
-            <artifactId>jcabi-manifests</artifactId>
-        </dependency>
+<!--         <dependency> -->
```

Comment:
@pecko do we really need to have this comments here? would it be better to remove these lines completely?